pkg/registry/apis/folders/legacy_storage.go

```diff
@@ -35,6 +36,7 @@ type legacyStorage struct {
 	service        folder.Service
 	namespacer     request.NamespaceMapper
 	tableConverter rest.TableConvertor
+	accessControl  accesscontrol.AccessControl
```

Comment:
This is not needed anymore right?

Reply:
Nice catch, removed it.

Thanks for the reviews!